modular_skyrat/modules/sec_haul/code/guns/cargo_stuff.dm

```diff
@@ -166,6 +167,7 @@
 		/obj/item/gun/ballistic/automatic/sol_rifle/machinegun = 1,
 		/obj/item/ammo_box/magazine/c40sol_rifle/drum = 2,
 	)
+*/ //BUBBER EDIT: IT'S AS BAD AS YOU THOUGHT
```

Comment:
Why? It's emag only already

Reply:
It's extremely powerful and is notably better than anything a traitor could possibly get, with significantly more ammo as well.